cinema/urls.py

```diff
@@ -13,9 +14,12 @@
 router.register("genres", GenreViewSet)
 router.register("actors", ActorViewSet)
 router.register("cinema_halls", CinemaHallViewSet)
-router.register("movies", MovieViewSet)
+router.register(r'movies', MovieViewSet, basename='movie')
 router.register("movie_sessions", MovieSessionViewSet)
+router.register(r"orders", OrderViewSet)
```

Comment:
Code Style: Consider using double quotes for consistency with the rest of the file, as other strings are using double quotes.